Live_data_update.sql

```diff
@@ -0,0 +1,10 @@
+https://trello.com/c/CKuDXAbq
+update cqc."Establishment" set "MainServiceFKValue"=20 where "EstablishmentID"=340;
+
+
+
+
+https://trello.com/c/oeYhp3Ui
+
+update cqc."Establishment" set "IsRegulated"='t' where "EstablishmentID"=340;
```

Comment:
`IsRegulated` is a boolean, so isn't it more prudent to update with primitive value `true` rather than text value `'t'`.

Regarding this Establishment with id 340, does it have a "LocationID"? If `IsRegulated` is true, then it needs to have a location id too.

Reply:
Hi Warren

Thanks , I will change text value to Boolean value and id 340 does have a locationID too , I will add that and send it again.

Thanks
Shakir

Reply:
Hi Warren I have updated and corrected the script.
There were 2 mistakes, 1-EstablishmentID corrected to 337 for IsRegulated and Boolean value set to True.

Reply:
Hi Warren 

I got location ID from Jackie and we are good to go with this change to the live database.

Before that, I would update the changes to UAT02 and verify it if Ann can see those changes the apply it to the production database.

Thanks
Shakir